spec/features/courses/index_spec.rb

```diff
@@ -74,7 +78,7 @@
     end
 
     scenario "it shows 'add a new course' link" do
-      expect(page).to have_link('Add a new course', href: Settings.google_forms.new_course_for_accredited_bodies_url)
+      expect(page).to have_link('Add a new course', href: /#{Settings.google_forms.new_course_for_accredited_bodies.url.gsub('?', '\?')}/)
```

Comment:
Should we make an effort to add these things into the page objects? It should be an easy change.